FaunaDB.Client/Types/Attributes.cs

```diff
@@ -50,10 +50,35 @@ public class FaunaFieldAttribute : Attribute
         /// </summary>
         public object DefaultValue { get; set; }
 
+        /// <summary>
+        /// Overrides the fauna type <see cref="FaunaDB.Types"/> namespace for
+        /// list of accepted values.
+        /// Type should inherit from <see cref="FaunaDB.Types.Value"/>.
+        /// Such as, <see cref="TimeV"/>, <see cref="DateV"/>, <see cref="DoubleV"/>, etc
+        ///
+        /// Not fully supprted, only use for the <see cref="Encoder"/> specifically to enforce
```

Comment:
I'm not liking too much of this, isn't better to create a dedicated attribute just for this? Like `FaunaDateAttribute/FaunaTimeAttribute`?